services/gateway/services/search.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Search API",
+  "api_id": "12",
+  "org_id": "default",
+  "definition": {
+    "location": "",
+    "key": ""
+  },
+  "use_keyless": true,
+  "auth": {
+    "auth_header_name": ""
+  },
+  "version_data": {
+    "not_versioned": true,
+    "versions": {
+      "Default": {
+        "name": "Default",
+        "expires": "3000-01-02 15:04"
+      }
+    }
+  },
+  "proxy": {
+    "listen_path": "/api/search",
+    "target_url": "http://search:8000/",
```

Comment:
For now it will prefix the search, so `/indexables` in the service will be `/api/search/indexables`. If this changes in the service we can change the target url to have `api/search` on the end of it